Trading/Exchange/default_spot_ccxt/default_spot_ccxt.py

```diff
@@ -0,0 +1,31 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_trading.enums import AccountTypes
+from octobot_trading.exchanges.implementations.spot_ccxt_exchange import SpotCCXTExchange
+
+
+class DefaultCCXTSpotExchange(SpotCCXTExchange):
```

Comment:
Used when an exchange has no specific tentacle ?

Reply:
Exactly. The idea is that user can disable or enable this exchange if they want to force the generic CCXT exchange or not